docs/source/Plugin/P053.rst

```diff
@@ -133,23 +133,23 @@ Select the option that shows the connected sensor model, to adjust the data retr
 Output
 ^^^^^^
 
-**Output values** When the PMS5003ST **Sensor model** is selected, extra output values are available. To choose which values are made available in the task, this combobox provides the available options
+**Output values** With most **Sensor model** options, except the PMS2003 and PMS3003 models, extra output values are available. To choose which values are made available in the task, this combobox provides the available options
 
 Available options:
 
 .. image:: P053_OutputValuesOptions.png
 
-*Particles: pm1.0, pm2.5, pm10* The default output values, available in all supported sensors.
+*Particles µg/m3: pm1.0, pm2.5, pm10* The default output values, available for all supported sensors.
 
-- PM1.0 - Concentration of particles 1um/m3
-- PM2.5 - Concentration of particles 2.5um/m3
-- PM10 - Concentration of particles 10um/m3
+- PM1.0 - Concentration of particles > 1 micron (µm/m3)
+- PM2.5 - Concentration of particles > 2.5 micron (µm/m3)
+- PM10 - Concentration of particles > 10 micron (µm/m3)

```

Comment:
Not sure if this is the correct unit of measure.
I don't think  a 1-dimensional length unit per volumetric unit even makes sense in whatever context.

Reply:
I've updated the unit of measure to Particles / volume unit (0.1L & m3).